src/components/Pantry/Pantry.jsx

```diff
@@ -74,22 +84,27 @@ export default function Pantry(props) {
     let itemIndex = userProducts.findIndex(item => item.name === itemName)
 
     let newProducts = [...userProducts]
-
+    setError("")
```

Comment:
nit: a helper function like `clearError()` might make the intent here more clear by providing a better name.